docs/python_docs/themes/mx-theme/mxtheme/header_top.html

```diff
@@ -1,6 +1,6 @@
 <header class="site-header" role="banner">
   <div class="wrapper">
-      <a class="site-title" rel="author" href="{{theme_relative_url}}"><img
```

Comment:
Is there a way to change `theme_relative_url` rather than hardcoding version 1.8?